packages/gui/src/components/nfts/gallery/NFTGallery.tsx

```diff
@@ -28,6 +28,8 @@ export const defaultCacheSizeLimit = 1024; /* MB */
 
 const maxNFTsPerPage = 200;
 
+const scrollPosition = new Map();
```

Comment:
No need for this to be a map.

```suggestion
let scrollPosition = 0;
```

Reply:
Another problem with this is that the scroll position is remembered across all wallet keys. So setting the scroll position in a gallery with hundreds of NFTs, and then switching to another wallet key, the NFT gallery would try to restore the scroll position from the prior wallet key's gallery.